dvd_DgrafX/Source/DiscVisionDeluxeUE/DfisX/Daero.cpp

```diff
@@ -71,15 +68,12 @@ void step_Daero (Throw_Container &active_throw, float step_time)
     """
     */
 
+  Eigen::Vector3d wind_temp = {0,0,0};
 
+  Eigen::Vector3d disc_air_velocity_vector = d_velocity - wind_temp;
 
-	Eigen::Vector3d disc_air_velocity_vector = d_velocity - get_global_wind_vel ();
-
-
-
-  d_forces.disc_velocity_unit_vector = get_unit_vector (disc_air_velocity_vector);
-  make_unit_vector (d_orientation);
-
+  d_forces.disc_velocity_unit_vector = get_unit_vector(disc_air_velocity_vector);
+  make_unit_vector(d_orientation);
```

Comment:
I have temporarily disabled some logging and wind setting in DfisX, I'll bring this stuff back in a more modular way when I do my next DfisX org branch.